Tasks/AppCenterDistributeV1/Tests/L0OneIpaPass.ts

```diff
@@ -4,6 +4,7 @@ import tmrm = require('vsts-task-lib/mock-run');
 import path = require('path');
 import fs = require('fs');
 import azureBlobUploadHelper = require('../azure-blob-upload-helper');
+import { basicSetup } from './TestHelpers';
```

Comment:
any chance to adjust it to lower-case notation to fit other imports?

Reply:
I aligned it with TestHelpers from V3. If renaming to lower case, it will lead to lots of unrelated refactoring which I am not fond of.

Reply:
ok, gochu